src/api/views.py

```diff
@@ -553,25 +574,78 @@ def post(self, request, repo_base, repo_name):
             type: string
             required: true
 
+
         """
         username = request.user.get_username()
         serializer = CardSerializer(username, repo_base, request)
         card_name = request.data['card_name']
         query = request.data['query']
-        serializer.create_card(repo_name, query, card_name)
-        card = serializer.describe_card(repo_name, card_name)
-        return Response(card, status=status.HTTP_201_CREATED)
+        res = serializer.create_card(repo_name, query, card_name)
+        return Response(res, status=status.HTTP_201_CREATED)
 
 
 class Card(APIView):
 
     def get(self, request, repo_base, repo_name, card_name):
```

Comment:
Cards need to offer text/csv format like Query does. That'll take changes to both Card and CardSerializer.